packages/editor-ui/src/views/NodeView.vue

```diff
@@ -1292,6 +1293,37 @@ export default mixins(
 					parameters: {},
 				};
 
+				const credentialPerType = nodeTypeData.credentials && nodeTypeData.credentials
```

Comment:
can we move this to its own function? 
for example `this.addDefaultCredential(newNodeData)`